.circleci/config.yml

```diff
@@ -31,8 +31,11 @@ commands:
   install-ansible:
     steps:
       - run:
-          name: Install Ansible
+          name: Install Ansible CLI
           command: sudo apt-add-repository --yes --update ppa:ansible/ansible && sudo apt install ansible
+      - run:
+          name: Install Ansible library
+          command: CRYPTOGRAPHY_DONT_BUILD_RUST=1 pip install ansible
```

Comment:
What's this CRYPTOGRAPH_DONT_BUILD_RUST about?

I'd prefer we just accept the cryptography guy actually has most of the world wrapped around his finger and we get used to installing rustc

Reply:
From the blurb I got in the console about this, it sounded to me like it disabling the building of Rust bindings for Ansible and not that it would be missing secure functionality. Let me confirm with one other source at least...

Reply:
Oh no, seems it comes from here and will eventually be a hard fail: https://cryptography.io/en/latest/faq.html. *sigh* Having to use Linux packages, pip, and a Rust compiler just to use Ansible? That's gotten a little silly IMO.

Reply:
Oh, this was a huge flamewar in the last few weeks.  Get some popcorn, read through https://github.com/pyca/cryptography/issues/5771

Reply:
side note, on ubuntu, I think it's likely you could just `pip update pip` to some minimum version, and you wouldn't have to build from scratch.

Reply:
Thanks for the nightmare reading material. Installing Rust for now...

Reply:
Having read more, I like the idea of upgrading pip. I'll put that on my list of things to start adding everywhere when convenient - for now I wanna get this unblocked.